crates/bermuda/src/lib.rs

```diff
@@ -145,6 +151,64 @@ fn numpy_polygons_to_rust_polygons(polygons: Vec<PyReadonlyArray2<'_, f32>>) ->
     polygons_
 }
 
+fn numpy_polygons_to_rust_polygons_3d(
+    polygons: Vec<PyReadonlyArray2<'_, f32>>,
+) -> (Vec<Vec<Point>>, usize, f32) {
+    let mut same_coordinates = [true; 3];
```

Comment:
Maybe `collinearity_axis` instead of `same_coordinates`

Same coordinates doesn't give me a good sense of what it is.

Reply:
changed to `is_collinearity_axis`